app/views/static_pages/help.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= provide :title, "Help" %>
+
+<h1>Help</h1>
+<p>
+ Get help on the Ruby on Rails Tutorial at the
+ <%= link_to "Rails Tutorial help page", "http://www.railstutorial.org/help" %>
+ To get help on this sample app, see the
+ <a href="http://www.railstutorial.org/book"><em>Ruby on Rails Tutorial</em>
```

Comment:
use link_to instead of a tag
https://apidock.com/rails/ActionView/Helpers/UrlHelper/link_to

Reply:
done